app/views/tutors/_form.html.erb

```diff
@@ -0,0 +1,118 @@
+
+  <% if @tutor.errors.any? %>
+    <div id="error_explanation">
+      <h2>
+        <%= pluralize(@tutor.errors.count, "error") %> prohibited
+        this tutor from being saved:
+      </h2>
+      <ul>
+        <% @tutor.errors.full_messages.each do |msg| %>
+          <li><%= msg %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="row">
+    <div class="col-md-3">
+      <%= f.input :first_name %>
+    </div>
+    <div class="col-md-4">
+      <%= f.input :last_name %>
+    </div>
+    <div class="col-md-3">
+      <%= f.input :dob, as: :date, label: "Date of Birth", selected: 1975,
+                                   start_year: Date.today.year - 70,
+                                   end_year: Date.today.year - 12 %>
+    </div>
+    <div class="col-md-2">
+      <div class="form-group">
+        <%= f.label "Gender" %><br>
+        <%= collection_radio_buttons(:tutor, :gender, [["male", "Male"] ,["female", "Female"]], :first, :last) do |b| %>
+          <label class="radio-inline">
+            <%= b.radio_button %> <%= b.text %>
+          </label>
+        <% end %>
+      </div>
+    </div>
+  </div>
+
+  <hr>
+
+  <div class="row">
+    <div class="col-md-5">
+      <%= f.input :address %>
+    </div>
+    <div class="col-md-3">
+      <%= f.input :city %>
+    </div>
+    <div class="col-md-1">
+      <%= f.input :state, selected: [ "MA" ], collection: us_states %>
+    </div>
+    <div class="col-md-2">
+      <%= f.input :zip %>
```

Comment:
Special elements like this can get a really easy form of front end "validation" of max char. Same with phone.